go/chat/utils/utils.go

```diff
@@ -313,7 +313,7 @@ type DebugLabeler struct {
 
 func NewDebugLabeler(log logger.Logger, label string, verbose bool) DebugLabeler {
 	return DebugLabeler{
-		log:     log,
+		log:     log.CloneWithAddedDepth(1),
```

Comment:
I tried this "just add depth once on the member logger" approach back in https://github.com/keybase/client/pull/9042, and I ran into trouble. I think the problem is that some of the logging calls (like `Trace` below) return a function that's supposed to be deferred in the caller. That means it ends up running in the caller's stackframe (I think), so we have to _avoid_ adding depth.

I ended up adding depth manually in each logging function, so that I could do it just for the direct logging bits, and not for the deferred bits.

Edit: Or am I totally wrong, based on your comment below?

Reply:
Ah, so you're the one that got rid of the `standard.go` lines! Thanks!

My understanding of defers and caller stacks is that the first frame is where the deferred function is defined, and the second is the line that triggers the defer, i.e. where the enclosing function returns. So therefore, deferred functions always need an additional depth of 1. And that doesn't seem to change even if the deferred function is not defined in the enclosing function.

Just to make sure I'm not crazy, I'll paste logs for before/after...